modules/ROOT/pages/configure-host.adoc

```diff
@@ -29,7 +27,7 @@ Your partners provide you with their own endpoint configurations to which your o
 * Your digital certificates, such as an X.509 certificate
 * Electronic Data Interchange (EDI) message type preferences for your organization
 * DataWeave transformation maps your organization uses for B2B transformations +
-Create these maps using Anypoint Studio before you use Partner Manager.
+Use Anypoint Studio to create the DataWeave transformation maps before you use Partner Manager.
```

Comment:
Will we have a See Also task topic ref. that tells how to do this, or does the audience know?